diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/CommentsFormattingTest.kt

```diff
@@ -0,0 +1,240 @@
+                    |       /* First comment */
+                    |       val first = 5 // Some comment
+                    |       
+                    |       /**
+                    |       * kDoc comment
+                    |       */
+                    |       val second = 6
+                    |       
+                    |       /**
+                    |       * asdasd
+                    |       */
+                    |       fun testFunc() {
+                    |           val a = 5 // Some Comment
+                    |           
+                    |           // Fun in fun Block
+                    |           val b = 6
+                    |       }
+                    |    }
+                    |}
+                """.trimMargin()
+
+        lintMethod(CommentsFormatting(), code)
+    }
+
+    @Test
+    @Tag(WarningNames.COMMENT_NEW_LINES)
+    fun `check file new line above comment good` () {
+        val code =
+                """
+                    |package org.cqfn.diktat.ruleset.chapter3
+                    |
+                    |// Some comment
+                    |class Example {
+                    |
+                    |}
+                    |
+                    |// Some comment 2
+                    |class AnotherExample {
+                    |
+                    |}
+                """.trimMargin()
+
+        lintMethod(CommentsFormatting(), code)
+    }
+
+    @Test
+    @Tag(WarningNames.COMMENT_NEW_LINES)
+    fun `check file new line above comment bad` () {
+        val code =
+                """
+                    |package org.cqfn.diktat.ruleset.chapter3
+                    |// Some comment
+                    |class Example {
+                    |
+                    |}
+                    |
+                    |// Some comment 2
+                    |class AnotherExample {
+                    |
+                    |}
+                """.trimMargin()
+
+        lintMethod(CommentsFormatting(), code,
+                LintError(2,1,ruleId, "${Warnings.COMMENT_NEW_LINES.warnText()} // Some comment", true))
+    }
+
+    @Test
+    @Tag(WarningNames.COMMENT_NEW_LINES)
+    fun `check file new line above comment bad - block and kDOC comments` () {
+        val code =
+                """
+                    |package org.cqfn.diktat.ruleset.chapter3
+                    |/* Some comment */
+                    |class Example {
+                    |
+                    |}
+                    |/**
+                    |* Some comment 2
+                    |*/
+                    |
+                    |class AnotherExample {
+                    |
+                    |}
+                """.trimMargin()
+
+        lintMethod(CommentsFormatting(), code,
+                LintError(2,1,ruleId, "${Warnings.COMMENT_NEW_LINES.warnText()} /* Some comment */", true),
+                LintError(6,1,ruleId, "${Warnings.COMMENT_NEW_LINES.warnText()} /**\n" +
```

Comment:
Please don't use mutliline string in LintErrors, it makes logs far less readable. Just use the first line of text, or first line with letters, if you want.